### DISCUSSION TITLE
Error installing BindFS

### DISCUSSION THREAD

**Discussion by martinmanon** [Homebrew/discussions#3560] [Casks]
Hello, 

I'm totally not familiar with Brew and trying to install BinFS. 
I've already insttalled fuse via this command brew install --cask macfuse (wich worked properly, I think)
When I try to install BindFS I get this error message :

Warning: No available formula with the name "pkg-config" (dependency of gromgit/fuse/bindfs-mac).
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.

Does someone have any idea of how to do ? 

Thank you

**Comment by SMillerDev** [comment]
What does `brew doctor` say?

**Reply by martinmanon** [reply]
I get this 
```console
Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences or run:
  softwareupdate --all --install --force

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/all/.
You should download the Command Line Tools for Xcode 12.4.


Warning: Homebrew/homebrew-core was not tapped properly! Run:
  rm -rf "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core"
  brew tap homebrew/core

Warning: Some frameworks can be picked up by CMake's build system and will likely
cause the build to fail. To compile CMake, you may wish to move these
out of the way:
  /Library/Frameworks/libcurl.framework

Warning: Some taps are not on the default git origin branch and may not receive
updates. If this is a surprise to you, check out the default branch with:
  git -C $(brew --repo homebrew/core) checkout master
```

**Reply by SMillerDev** [reply]
```
Warning: Homebrew/homebrew-core was not tapped properly! Run:
rm -rf "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core"
brew tap homebrew/core
```

These instructions will resolve your issue

**Reply by martinmanon** [reply]
I run the command but I get  several lines like this : 
rm: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/.git/objects/2e: Directory not empty
rm: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/.git/objects/13/a572b70c145e9e20d9ecc4486f8557fedffbd8: Permission denied
rm: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/.git/objects/13: Directory not empty
rm: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/.git/objects: Directory not empty
rm: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/.git: Directory not empty
rm: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core: Directory not empty

**Reply by SMillerDev** [reply]
That means your user is not the owner of your homebrew install, you can try and fix it with `sudo chown -R $(whoami) /usr/local/*`
